CHANGELOG.rst

```diff
@@ -7,8 +7,73 @@ Synapse Changelog
 Unreleased - YYYY-MM-DD
 =======================
 
+Automatic Migrations
+--------------------
+- Migrate the ``file:bytes:mime:pe:imphash`` from a ``guid`` to a ``hash:md5``
```

Comment:
Should add 'property' after ``file:bytes:mime:pe:imphash`` or remove the 'the' before it